docs/api/web-contents.md

```diff
@@ -1909,3 +1909,7 @@ A [`Debugger`](debugger.md) instance for this webContents.
 
 A `Boolean` property that determines whether or not this WebContents will throttle animations and timers
 when the page becomes backgrounded. This also affects the Page Visibility API.
+
+#### `contents.webFrame` _Readonly_
```

Comment:
Perhaps this should be `mainFrame`?

Reply:
I originally was using this and changed it for some reason. I'll change it back since I think it improves semantics.

Reply:
9015ba1